README.md

```diff
@@ -73,7 +73,7 @@ WifiWizard2.scan([options])
 For functionality, you need to note the following:
  - Connect/Disconnect only works for iOS11+
  - Can't run in the simulator so you need to attach an actual device when building with xCode
- - Need to add the 'HotspotConfiguration' and 'NetworkExtensions' capabilities to your xCode project
+ - Will add the 'HotspotConfiguration' and 'NetworkExtensions' capabilities to your xCode project
```

Comment:
I wouldn't say that it 'Will Add' the functionality. Build processes can screw up so I would recommend we change the ReadMe to say:
'Ensure the 'HotspotConfiguration' and 'NetworkExtensions' capabilities are added to your xCode project
 - The plugin should automatically add these capabilities however double check prior to building'